OH-Archive/publication.html

```diff
@@ -14,10 +14,12 @@
 <div id='id-Saisonnières-und-Saisonniers' class="container" style="padding-top: 15px; text-align: left">
     <h3 style="text-align: left;">Kontext zu Saisonnières und Saisonniers</h3>
     <div style="font-size: 20px;">
-        Seminar: «Wir, Saisonniers und Saisonnières…» Für eine Ausstellung im NMB (Neues Museum Biel) Oral History Interviews mit ehemaligen Saisonniers und Saisonnières durchführen und einen Podcast produzieren </div>
-    <div id='id-Saisonnières-und-Saisonniers-links'>
-        <a href="https://www.ksl.unibe.ch/KSL/kurzansicht?9&stammNr=472151&semester=HS2021&lfdNr=0"  target="_blank" class="link-primary" style="font-size: 20px;">Kursbeschreibung</a>
+        Das Neue Museum Biel wird 2022 die Arbeits- und Lebensbedingungen der Saisonniers und Saisonnières beleuchten. in diesem Zusammenhang werden wir in unserem Seminar Oral History Interviews mit Arbeiter*innen führen, die unter dem Saisonnier-Statut in der Region Biel arbeiteten. Anschliessend werden wir auf der Grundlage dieser Interviews Podcasts als Begleitmaterial zur Ausstellung erstellen.
```

Comment:
add wrapping paragraphs to make the text more legible.
```suggestion
        <p>Das Neue Museum Biel wird 2022 die Arbeits- und Lebensbedingungen der Saisonniers und Saisonnières beleuchten. in diesem Zusammenhang werden wir in unserem Seminar Oral History Interviews mit Arbeiter*innen führen, die unter dem Saisonnier-Statut in der Region Biel arbeiteten. Anschliessend werden wir auf der Grundlage dieser Interviews Podcasts als Begleitmaterial zur Ausstellung erstellen.</p>
```

Reply:
Done